### PR TITLE
Add entrypoint

### PR DESCRIPTION
**Pull request by iTitus**

### CODE REVIEW

File: whist/server/cli.py [hunk 1]
```diff
@@ -1,8 +1,12 @@
...
-
+"""CLI entrypoint"""
 import uvicorn
 
 from whist.server import app
 
-if __name__ == '__main__':
+
+def main():
     uvicorn.run(app, host='0.0.0.0', port=8000)
```
Comment (Segelzwerg) [inline whist/server/cli.py:8]: Intend missing?

**Review by Segelzwerg — CHANGES_REQUESTED** [review]

File: whist/server/cli.py (hunk 1 above)
Comment (iTitus) [reply whist/server/cli.py:8]: Nothing is missing here. the `if __name__ == '__main__'` got moved

Comment (Segelzwerg) [reply whist/server/cli.py:8]: In the diff it looks like the body of `main()` has no indent

Comment (iTitus) [reply whist/server/cli.py:9]: nope

Comment (Segelzwerg) [reply whist/server/cli.py:9]: Weird, it was only on the mobile ...

**Review by Segelzwerg — APPROVED** [review]

**Pull request merged**